### PR TITLE
Adding a note in performance related repr() of masked column

### PR DESCRIPTION
**Pull request by himanshupathak21061998**
Fix:#8453
I added a note in the performance section for this.

### CODE REVIEW

File: docs/table/performance.inc.rst [hunk 1]
```diff
@@ -24,3 +24,17 @@ end::
   >>> for i in range(100):
   ...    rows.append((1, 2))
   >>> t = Table(rows=rows, names=['a', 'b'])
+
+Writing a |Table| with |MaskedColumn| to .ecsv using
```
Comment (pllim) [inline docs/table/performance.inc.rst]:
```suggestion
Writing a |Table| with |MaskedColumn| to ``.ecsv`` using
```

File: docs/table/performance.inc.rst [hunk 2]
```diff
@@ -24,3 +24,17 @@ end::
...
+:meth:`~astropy.table.Table.write` can be very slow::
+
+    >>> from astropy.table import Table
+    >>> import numpy as np
+    >>> x = np.arange(10000, dtype=float)
+    >>> tm = Table([x], masked=True)
+    >>> tm.write('tm.ecsv', overwrite=True)
+
+If you want to write .ecsv using Table.write then use serialize_method='data_mask'.
```
Comment (pllim) [inline docs/table/performance.inc.rst]:
```suggestion
If you want to write ``.ecsv`` using :meth:`~astropy.table.Table.write`,
then use ``serialize_method='data_mask'``.
```

File: docs/table/performance.inc.rst [hunk 3]
```diff
@@ -24,3 +24,17 @@ end::
...
   >>> t = Table(rows=rows, names=['a', 'b'])
+
+Writing a |Table| with |MaskedColumn| to .ecsv using
+:meth:`~astropy.table.Table.write` can be very slow::
+
+    >>> from astropy.table import Table
+    >>> import numpy as np
+    >>> x = np.arange(10000, dtype=float)
+    >>> tm = Table([x], masked=True)
+    >>> tm.write('tm.ecsv', overwrite=True)
```
Comment (pllim) [inline docs/table/performance.inc.rst]: Should we mark this line as SKIP to not waste CI resources on something that is known to be slow?

File: docs/table/performance.inc.rst [hunk 4]
```diff
@@ -24,3 +24,17 @@ end::
...
+
+If you want to write .ecsv using Table.write then use serialize_method='data_mask'.
+It uses the non-masked version of data and it is faster.
+
+    >>> tm.write('tm.ecsv', overwrite=True, serialize_method='data_mask')
```
Comment (pllim) [inline docs/table/performance.inc.rst]: This needs to be skipped if `yaml` is not installed. You can look at other existing documentation for example on how to accomplish this.
```
ImportError: No module named 'yaml'
```

Comment (himanshupathak21061998) [reply docs/table/performance.inc.rst]: Skipping this in doctest as other examples did same

**Comment by himanshupathak21061998** [comment]
@pllim Thanks for helping

**Comment by himanshupathak21061998** [comment]
Thanks for merging.

**Pull request merged**